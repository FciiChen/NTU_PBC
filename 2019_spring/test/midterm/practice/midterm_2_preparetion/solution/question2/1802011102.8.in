hnrPsq2Hb, Nydt,Aj1
83me1, HqDyvMIxbA,JzQHfWgk8
n4fLrU4z8y, 9ZoYOYJ9,cDhzhPxz
0QmpBCk, DnTceE,wpD
C7o9ZMR, WU1KU,eMmfadwc8
RLtlmq, ABXnHMwHK,yron
sIB, G9OJJkL6,lLnhwDxZd
ckOyimk, uGBZXfIOF,aqVUZ4m
eaGZ3, dufbw9uh59,pxLRN
BREAK
ckOyimk ,  1gtZA
RLtlmq ,  XU5FoAP2n
0QmpBCk ,  91eJ
n4fLrU4z8y ,  UaPU
BREAK
