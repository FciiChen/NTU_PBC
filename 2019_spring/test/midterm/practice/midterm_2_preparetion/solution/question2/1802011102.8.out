hnrPsq2Hb  Nydt       Aj1        NULL      
83me1      HqDyvMIxbA JzQHfWgk8  NULL      
n4fLrU4z8y 9ZoYOYJ9   cDhzhPxz   UaPU      
0QmpBCk    DnTceE     wpD        91eJ      
C7o9ZMR    WU1KU      eMmfadwc8  NULL      
RLtlmq     ABXnHMwHK  yron       XU5FoAP2n 
sIB        G9OJJkL6   lLnhwDxZd  NULL      
ckOyimk    uGBZXfIOF  aqVUZ4m    1gtZA     
eaGZ3      dufbw9uh59 pxLRN      NULL      
